presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/StreamPropertyDerivations.java

```diff
@@ -450,6 +451,13 @@ public StreamProperties visitTableWriter(TableWriterNode node, List<StreamProper
             return properties.withUnspecifiedPartitioning();
         }
 
+        @Override
+        public StreamProperties visitUpdate(UpdateNode node, List<StreamProperties> inputProperties)
+        {
+            StreamProperties properties = Iterables.getOnlyElement(inputProperties);
```

Comment:
nit: static import for `getOnlyElement`?

Reply:
There are many instances of `Iterables.getOnlyElement` in the file already. I would prefer to keep the diff small